components/page/header/mega-menu/custom/absch.vue

```diff
@@ -2,13 +2,27 @@
     <div class="col-12 text-wrap px-0">
         <LazyPageHeaderMegaMenuHeader  :menu="menu" />
 
+        <section v-for="(aChild,j) in drupalMenus" :key="j">
+            <p >
+                <LazyPageHeaderMegaMenuLink :title="aChild.title"  :menu="aChild" />
+            </p>
+        </section>
         <LazyPageHeaderMegaMenuLink v-for="(aMenu,j) in menu.children" :key="j" :menu="aMenu" />
     </div>
 </template>
 <script setup>
+    import clone from 'lodash.clonedeep';
```

Comment:
I wonder how better it is to have each function imported and compiled independantly than importing the whole library once